### ISSUE TITLE
[$75] Keyboard Navigation: Landing Page: The social network sharing links are not focusable with Tab key

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#74512]
## Action Performed
1. Using any desktop browser, navigate to https://new.expensify.com/
2. Press Tab key and try to focus the social media links, located in the footer under 'Learn More'

## Expected Result
The keyboard user should be able to navigate using the keyboard to the social media links, Podcast, Twitter, Instagram, Facebook, LinkedIn, and activate them without the use of the mouse

## Actual Result
When users navigate to the social network sharing links via keyboard navigation, they are not focusable with Tab key. Focus moves from 'Investor relations', skips the social network links, and moves onto 'Create a new account'

HTML:
<a href=""[https://www.instagram.com/expensify""](https://www.instagram.com/expensify%22%22) data-tag=""pressable"" tabindex=""-1"" class=""css-g5y9jx r-1otgn73 r-1loqt21 r-1d4mawv r-14gqq1x"" data-component=""GenericPressable"" data-element=""Pressable"" data-source-file=""BaseGenericPressable.tsx""><div class=""css-g5y9jx"" data-testid=""""><svg xmlns=""[http://www.w3.org/2000/svg""](http://www.w3.org/2000/svg%22%22) xml:space=""preserve"" viewBox=""0 0 20 20"" width=""24"" height=""24"" hovered=""false"" pressed=""false"" fill=""#E7ECE9""><path d=""...""></path></svg></div></a>"

## Failed Checkpoint
WCAG 2.1.1 Keyboard
https://www.w3.org/WAI/WCAG22/Understanding/keyboard

## Area issue was found in:
Landing Page

## Failed WCAG checkpoints
2.1.1: Keyboard

## User Impact
People who are blind cannot use pointing devices like a mouse that require eye-hand coordination. People with low vision may have trouble finding or tracking a pointer indicator on screen. Some people with hand tremors find using a mouse very difficult and therefore usually use a keyboard.

## Suggested Resolution
Remove the currently used tabindex="-1" from the links, so they are focusable with Tab key.
https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Global_attributes/tabindex

## Environment(s)
Windows Windows 10 - Chrome

<img width="1918" height="1004" alt="Image" src="https://github.com/user-attachments/assets/88bb7969-5c7d-4601-b2dc-1b51d7af01a2" />

https://github.com/user-attachments/assets/dacf9102-ba80-40ad-8726-ea49e4d578d3

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6985815
**Issue reported by:** NA
 **Slack conversation:** NA

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~022009685721729707540</li>
        <li>Upwork Job ID: 2009685721729707540</li>
        <li>Last Price Increase: 2026-01-09</li>
<li>Automatic offers: </li>
<ul>
<li>daledah | Contributor | 110091772</li>
</ul></li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @roryabraham</details>

**Comment by mallenexpensify** [comment]
**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** NA
**Issue reported by:** NA
**Slack conversation:** NA

**Comment by rushatgabhane** [comment]
the correct amount for this issue is $75
